articles/application-insights/app-insights-ip-addresses.md

```diff
@@ -199,3 +199,12 @@ Open ports 80 (http) and 443 (https) for incoming traffic from these addresses:
 | --- | --- | --- | --- |
 | Agent | agent.azureserviceprofiler.net | dynamic | 443
 | Portal | gateway.azureserviceprofiler.net | dynamic | 443
+| Storage | core.windows.net | dynamic | 443
+
+## Snapshot Debugger
+
+| Purpose | URI | IP | Ports |
+| --- | --- | --- | --- |
+| Agent | ppe.azureserviceprofiler.net | dynamic | 443
+| Portal | ppe.gateway.azureserviceprofiler.net | dynamic | 443
```

Comment:
It's not needed at snapshot collection time. It's needed to view the snapshot via the Portal.